apps/condo/pages/contact/[id]/index.tsx

```diff
@@ -15,7 +15,7 @@ import { FrontLayerContainer } from '@condo/domains/common/components/FrontLayer
 import { NotDefinedField } from '@condo/domains/user/components/NotDefinedField'
 import { Button } from '@condo/domains/common/components/Button'
 import { useOrganization } from '@core/next/organization'
-import  { TicketCard } from '@condo/domains/common/components/TicketCard/TicketCard'
+import  { TicketCardList } from '@condo/domains/common/components/TicketCard/TicketCardList'
```

Comment:
If this component is not used elsewhere - better to move it inside the domain "contact".